### PR TITLE
DOC: added copy functionality in code blocks

### PR DESCRIPTION
**Pull request by meghanacosmos**
Hi, I was going through the docs and thought it would be convenient to have a copy button in the code blocks. So I tried to add it and it works fine. If this is something you'd want as well then, I'll continue to work on this further to make it more reusable and clean. Please let me know what you think. Thank you. 

(I've downloaded the icon from https://uxwing.com/copy-icon/, which states that _" License: All icons are free to use any personal and commercial projects without any attribution or credit. "_  )

- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by mroeschke** [comment]
> Sure, would you want me to give `sphinx-copybutton` a try?

Sure, that would be great.

File: doc/source/conf.py [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "matplotlib.sphinxext.plot_directive",
     "sphinx.ext.intersphinx",
     "sphinx.ext.coverage",
+    "sphinx_copybutton",  # helps add a little “copy” button to the code blocks
```
Comment (mroeschke) [inline doc/source/conf.py:64]: I don't think the comment adds much to this line (I think `copybutton` is fairly clear)

**Comment by Meghana-12** [comment]
Do I have to add the `sphinx-copybutton` to dependencies in the `environment.yml` file as well?

**Comment by jreback** [comment]
> Do I have to add the `sphinx-copybutton` to dependencies in the `environment.yml` file as well?

yes exactly

**Comment by datapythonista** [comment]
@Meghana-12 do you have time to add the missing dependency, and build the docs to see everything is working as expected?

**Comment by mroeschke** [comment]
Thanks for the pull request, but it appears to have gone stale. If interested in continuing, please merge in the main branch, address any review comments and/or failing tests, and we can reopen.

If you need assistance in building the documentation, here's a guide: https://pandas.pydata.org/docs/development/contributing_documentation.html#how-to-build-the-pandas-documentation

**Pull request closed without merging**